core/lib/multivm/src/versions/vm_latest/tracers/refunds.rs

```diff
@@ -146,6 +144,15 @@ impl<S> RefundsTracer<S> {
             U256::zero()
         });
 
+        tracing::trace!(
+            "Fee benchmark for transaction with hash {}",
+            hex::encode(tx_hash.as_bytes())
+        );
+        tracing::trace!("Gas Limit: {}", tx_gas_limit);
+        tracing::trace!("Gas spent on computation: {}", gas_spent_on_computation);
```

Comment:
I wonder we we could persist some of this inside the execution_info json in DB. (this would make debugging a lot easier)

Reply:
cc @perekopskiy WDYT? Obviously out of scope of the PR

Reply:
Yes, I think it's a good idea. Shouldn't be that hard to save it